### ISSUE TITLE
Slider's increase and decrease buttons should not be focusable

### ISSUE THREAD

**Issue by martintabakov** [telerik/kendo-ui-core#7818]
### Bug report
When the Slider's increase and decrease buttons should not be focusable.

### Reproduction of the problem
1. Open the Keyboard Navigation demo - https://demos.telerik.com/kendo-ui/slider/keyboard-navigation
2. Focus the Decrease button
3. Try pressing space/enter key

### Current behavior
The Slider's value cannot be changed through the keyboard when the buttons are focused

### Expected/desired behavior
Increase and Decrease buttons should not be focusable. They should have tabindex -1 and aria-hidden attribute, following the [Slider aria spec](https://github.com/telerik/web-components-ux/blob/develop/docs/components/slider/aria.json). Slider's keyboard navigation is described in the [Keyboard spec](https://github.com/telerik/web-components-ux/blob/develop/docs/components/slider/keyboard-navigation.json).
Clicking on Increase/Decrease button should focus the draghandle of the Slider

### Environment
* **Kendo UI version:** 2024.1.319 
* **Browser:** [all]
